deepchecks/vision/datasets/detection/yolo_to_coco.py

```diff
@@ -0,0 +1,137 @@
+import json
+import os
+import os.path as osp
+from typing import Sequence, Optional, Union
+import datetime
+import cv2
+import numpy as np
+import uuid
+
+
+YOLO_PATH = "/Users/nirbenzvi/code/DeepChecks/coco128"
+# Complete this by putting COCO labels
+CATEGORIES = ("person", "bicycle", "car", "motorcycle", "airplane", "bus",
+              "train", "truck", "boat", "traffic light", "fire hydrant",
+              "stop sign", "parking meter", "bench", "bird", "cat", "dog",
+              "horse", "sheep", "cow", "elephant", "bear", "zebra", "giraffe",
+              "backpack", "umbrella", "handbag", "tie", "suitcase", "frisbee",
+              "skis", "snowboard", "sports ball", "kite", "baseball bat",
+              "baseball glove", "skateboard", "surfboard", "tennis racket",
+              "bottle", "wine glass", "cup", "fork", "knife", "spoon", "bowl",
+              "banana", "apple", "sandwich", "orange", "broccoli", "carrot",
+              "hot dog", "pizza", "donut", "cake", "chair", "couch",
+              "potted plant", "bed", "dining table", "toilet", "tv", "laptop",
+              "mouse", "remote", "keyboard", "cell phone", "microwave",
+              "oven", "toaster", "sink", "refrigerator", "book", "clock",
+              "vase", "scissors", "teddy bear", "hair drier", "toothbrush")
+
+
+class YoloParser:
```

Comment:
Doesn't serve any purpose RN. Should perhaps be kept aside and added later to utils for users wishing to convert coco json format to our accepted format.